apple/internal/run_support.bzl

```diff
@@ -37,7 +37,25 @@ def _register_simulator_executable(ctx, output):
         },
     )
 
+def _register_app_executable(ctx, output):
```

Comment:
A nit about your nit :)

Should it be _register_macos_executable since the result of this function is writing the executable file and it’s consistent with the sim executable method in the same file?

Reply:
sounds good 👍